TradeItIosTicketSDK2/TradeItWebViewController.swift

```diff
@@ -0,0 +1,21 @@
+import UIKit
+
+class TradeItWebViewController: UIViewController, UIWebViewDelegate {
+
+    @IBOutlet weak var webView: UIWebView!
+    var url = ""
+    var pageTitle = ""
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.navigationItem.title = "Loading...";
+        self.webView.loadRequest(URLRequest(url: URL (string: self.url)!))
```

Comment:
We should not force unwrap here. We should `guard` instead and do nothing / dismiss the web view.